lib/rdoc/generator/template/rails/resources/js/searchdoc.js

```diff
@@ -234,7 +234,7 @@ Searchdoc.Panel.prototype = $.extend({}, Searchdoc.Navigation, new function() {
         if (window.location.protocol === "file:") {
```

Comment:
I think this case can be removed now this is implemented in Turbo, similar to https://github.com/rails/rails/pull/45932#discussion_r961942345